app/routes/bestuurseenheid/functionarissen.js

```diff
@@ -9,7 +9,8 @@ export default Route.extend(DataTableRouteMixin, {
   mergeQueryOptions(params) {
     this.set('bestuurseenheidId', params.bestuureenheid_id);
     return {
-      'filter[bekleedt][bevat-in][is-tijdsspecialisatie-van][bestuurseenheid][:id:]': params.bestuureenheid_id
+      'filter[bekleedt][bevat-in][is-tijdsspecialisatie-van][bestuurseenheid][:id:]': params.bestuureenheid_id,
+      'filter[:gte:einde]': new Date().toISOString()
```

Comment:
This won't work for einde: undefined.
Unfortunately, you can not ask mu-cl-resources for undefined values, so you will have to come up with a slightly more complex filter. 
Also, best to test your filter too. Here is an example call, 

```
/functionarissen?filter[bekleedt][bevat-in][is-tijdsspecialisatie-van][bestuurseenheid][:id:]=974816591f269bb7d74aa1720922651529f3d3b2a787f5c60b73e5a0384950a4&sort=is-bestuurlijke-alias-van.achternaam
```